### PR TITLE
syncthing-inotify 0.6.7

### PR DESCRIPTION
**Pull request by dvorakluk**
Updated to the newest version and some refactoring.

### CODE REVIEW

File: Library/Formula/syncthing-inotify.rb [hunk 1]
```diff
@@ -15,22 +15,11 @@ class SyncthingInotify < Formula
...
-      :revision => "f06b1e3b795091f2e1414067b08e5f07332cdb05"   # not sure !
-  end
-
   def install
     ENV["GOPATH"] = buildpath
-    ENV.append_path "PATH", buildpath
     bin_name = "syncthing-inotify"
-    Language::Go.stage_deps resources, buildpath/"src"
-    system "go", "build", "-o", bin_name
+    system "go", "get", "-d"
```
Comment (DomT4) [inline Library/Formula/syncthing-inotify.rb]: Please keep using our system of `go_resources` here. We don't use `go get` directly in formulae.

**Comment by dvorakluk** [comment]
Apologies, it looked better to me

**Comment by dunn** [comment]
> Should I also update the go_resource's revision? One of them isn't updated at all, but the another has 3 new commits.

Sure! (Assuming nothing breaks.)

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @dvorakluk; we appreciate it! :tada:

**Pull request closed without merging**